docs/editor/tasks.md

```diff
@@ -480,24 +480,24 @@ Below is an example of a custom task configuration that passes the current opene
 }
 ```
 
-Similarly, you can reference your project's configuration settings by prefixing the name with **${config:**. For example, `${config:python.pythonPath}` returns the Python extension setting `pythonPath`.
+Similarly, you can reference your project's configuration settings by prefixing the name with **${config:**. For example, `${config:python.formatting.autopep8Path}` returns the Python extension setting `formatting.autopep8Path`.
```

Comment:
Maybe there's a better example from a different extension, but the point is that ${config:python.pythonPath} won't work anymore, and instead users should use ${command:python.interpreterPath}